jupyter/Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN export DEBIAN_FRONTEND=noninteractive && apt-get update -y && apt-get -y ins
 RUN pip install --no-cache-dir librosa tqdm altair appdirs ipywidgets
 RUN python3 -m jupyter labextension install @jupyter-widgets/jupyterlab-manager
 RUN pip install git+https://github.com/jolibrain/dd_widgets
-RUN pip install git+https://github.com/deepdetect.git#egg=dd_client&subdirectory=clients/python
+RUN pip install "git+https://github.com/jolibrain/deepdetect.git#egg=deepdetect&subdirectory=clients/python"
```

Comment:
The egg should be the dd_client.

Reply:
```
> pip install "git+https://github.com/jolibrain/deepdetect.git#egg=dd_client&subdirectory=clients/python"

Defaulting to user installation because normal site-packages is not writeable
Collecting dd_client
  Cloning https://github.com/jolibrain/deepdetect.git to /tmp/pip-install-0ysoy0rw/dd-client_6cfac5fcfd904092afd24429f82d5356
  Running command git clone -q https://github.com/jolibrain/deepdetect.git /tmp/pip-install-0ysoy0rw/dd-client_6cfac5fcfd904092afd24429f82d5356
  WARNING: Generating metadata for package dd-client produced metadata for project name deepdetect. Fix your #egg=dd-client fragments.
WARNING: Discarding git+https://github.com/jolibrain/deepdetect.git#egg=dd_client&subdirectory=clients/python. Requested deepdetect from git+https://github.com/jolibrain/deepdetect.git#egg=dd_client&subdirectory=clients/python has inconsistent name: filename has 'dd-client', but metadata has 'deepdetect'
ERROR: Could not find a version that satisfies the requirement dd-client (unavailable)
ERROR: No matching distribution found for dd-client (unavailable)
```
With egg=deepdetect it works fine on my machine

In file https://github.com/jolibrain/deepdetect/blob/master/clients/python/setup.cfg, project name is deepdetect, not dd_client

Reply:
Ok, the package name here: https://github.com/jolibrain/deepdetect/blob/master/clients/python/setup.cfg is wrong.

It's recommended to have the same for root module name and package name.

Reply:
Changed it in deepdetect https://github.com/jolibrain/deepdetect/pull/1183 and here